### ISSUE TITLE
p-panel and p-inputNumber Accessibility Issues

### ISSUE THREAD

**Issue by jongmin-r-kim** [primefaces/primeng#18962]
### Describe the bug

If a p-inputNumber element is present within a collapsed p-panel element, upon entering the page we result in the following accessibility issue:
  `Element "input" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'`

This violation can be viewed via the  IBM Equal Accessibility Checker. 

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-6s3ve5nm?file=src%2Fapp%2FinputNumber.component.html

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker ](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the error: Element "input" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'

### Expected behavior

No accessibility violations on the p-inputNumber are present. It should provide a parameter with which to indicate that it is hidden upon entering, or otherwise avoid the accessibility violation
